course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -105,12 +105,24 @@
                 });
             };
 
+            $scope.getCourseDescription = function(course) {
+                // If a course's title is [NULL], attempt to display the short title.
+                // If the short title is also [NULL], display [School] 'Untitled Course' [Term Display]
+                if(typeof course.title != "undefined" && course.title.length > 0){
+                    return course.title;
+                }
+                else if(typeof course.short_title != "undefined" && course.short_title.length > 0){
+                    return course.short_title;
+                }
+                return 'Untitled Course';
```

Comment:
Hmm, yeah, good point - that's confusing! [Term Display] isn't quite what's exactly the next column over, but that info is all there in the table, right? So you're probably right, we don't need that stuff -- worth double-checking (and updating any test descriptions, and maybe specifying that in the ticket for clarity). Thanks!